### ISSUE TITLE
Change h1 from text to heading on 404 error/page not found page

### ISSUE THREAD

**Issue by eelzi-ONRR** [ONRR/onrr.gov-site#2376]
Right now the H1 on the 404 error page is tagged as text, not a heading. This is causing it fail to accessibility checkers and is prompting SEO issue reports in Bing Webmaster and Googls Search Console, since any failed redirect link points to the 404 page.

[https://www.onrr.gov/404](https://www.onrr.gov/404)

**Comment by Maroyafaied** [comment]
![image.png](https://images.zenhubusercontent.com/5c58acad07cc4d6f39e54f67/bfee8473-01be-4a0c-b17c-1894e7651f5e)
